main.py

```diff
@@ -19,36 +19,23 @@ def get_db() -> Session:
         db.close()
 
 
-@app.get("/")
-def read_root():
-    return {"Hello": "World"}
-
-
 @app.get("/books/", response_model=Page[pagination.BookOut])
 def get_books_list(
         author_id: int | None = None,
         db: Session = Depends(get_db)
 ):
-    return paginate(crud.get_books(author_id, db))
-
-
-def get_book_by_title(title: str, db: Session = Depends(get_db)):
-    return (
-        db.query(Book)
-        .filter(Book.title == title)
-        .first()
-    )
+    return paginate(crud.get_books(author_id=author_id, db=db))
 
 
 @app.post("/books/", response_model=schemas.BookRead)
 def create_book(book: schemas.BookCreate, db: Session = Depends(get_db)):
-    if get_book_by_title(book.title, db):
+    if crud.get_book_by_title(db, book.title):
         raise HTTPException(
             status_code=400,
             detail="Book with that title already exists",
         )
 
-    return crud.create_book(db, book)
+    return crud.create_book(db=db, book=book)
```

Comment:
no need to pass parameters' name explicitly, just reorder them so that it is in the same order that in function